xarray/tests/test_sparse.py

```diff
@@ -854,7 +855,10 @@ def test_sparse_coords(self):
         )
 
 
-@pytest.mark.xfail(reason="https://github.com/pydata/xarray/issues/5654")
+@pytest.mark.xfail(
+    LooseVersion(sparse.__version__) < "0.13.0",
```

Comment:
```suggestion
    sparse_version < "0.13.0",
```

You can import the sparse version from `from xarray.core.pycompat import sparse_version`.

Reply:
Ah great, thanks for pointing that out